frontend/src/components/dashboard/aliases/AddressPickerModal.test.tsx

```diff
@@ -0,0 +1,85 @@
+import { getAddressValidationMessage } from "./AddressPickerModal";
+
+describe("getAddressValidationMessage", () => {
+  it("returns `null` for valid addresses", () => {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const mockL10n = { getString: jest.fn() } as any;
+    expect(getAddressValidationMessage("valid-address", mockL10n)).toBeNull();
+  });
+
+  it("returns `null` for valid single-character addresses", () => {
```

Comment:
Hehe, I added this test because I initially made a mistake in the regex that disallowed valid single-character addresses, so this is to avoid a regression of that mistake in case someone in the future things they can simply the regex :)